docs/changelog.md

```diff
@@ -4,6 +4,18 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.5.2
+
+- FOREACH clause using outer variable scope to allow nesting. [#725](https://github.com/memgraph/memgraph/pull/725)
+- Add possibility to set and get property from node and relationship when using C++ API. [#732](https://github.com/memgraph/memgraph/pull/732)
+- FOREACH clause uses indexes in its children clauses [#736](https://github.com/memgraph/memgraph/pull/736)
```

Comment:
What's new here? Group new for each abilities together :) 
What does this enable?

Reply:
@brunos252